test/nn/test_embedding.py

```diff
@@ -1002,9 +1002,9 @@ def _test_EmbeddingBag_vs_Embedding(self, N, D, B, L, max_norm=None,
 
         output.backward(grad_output)
         ref_output.backward(grad_output)
-        es_weight_grad = es.weight.grad.data
+        es_weight_grad = torch.clone(es.weight.grad)
```

Comment:
Why did we need the `.data` here initially? 

From running locally, I think just `es.weight.grad` might be ok

Reply:
For more context, `.data` calls into `t.variable_data`, impl here 

https://github.com/pytorch/pytorch/blob/aa36821615a4b5cfa748b0fe07fb2f1050af629b/torch/csrc/autograd/variable.cpp#L438-L446

It seems like we just compare `es_weight_grad` with `e.weight.grad` below, so I don't see why this initial `.data` call here would be necessary 🤔